STM32/Util/Src/systeminfo.c

```diff
@@ -111,6 +120,43 @@ const char* systemInfo()
     return buf;
 }
 
+const char* statusInfo()
+{
+    static char buf[600] = { 0 };
+    int len = 0;
+    if (!(BS.boardStatus & BS_ERROR_Msk))
+    {
+        len += snprintf(&buf[len], sizeof(buf) - len, "Board Status: The board is operating normally.\r\n");
+        return buf;
+    } 
+    len += snprintf(&buf[len], sizeof(buf) - len, "Board Status: ");
```

Comment:
I noticed earlier that if there is an error there is no newline here. Perhaps add a newline to the message? Or make this an if/else and have another message `Board Status: There is an error.\r\n`